[5.0] Various post-release cleanups

dirkmueller opened the pull request.

dirkmueller committed:
install: stop creating PTF dirs for 12.2

Those were only needed for SOC7, so it shouldn't be necessary
for new installs.

dirkmueller committed:
install-chef-suse: Give a hint to the user when bootstrap wasn't run

While it isn't entirely clear to my why we separated bootstrap
from install, the error handling of missing bootstrap is suboptimal.

Detect the case that bootstrap failed prior trying to continue
with the installation, as it will never succeed and the error messages
are very hard to understand for the occassional user.

(cherry picked from commit c99b83262936e4bccb832a47f921c5c66a7b6a1f)

toabctl reviewed (changes requested):
Failing with:
```17:44:59 === 2019-03-05 16:44:50 +0000: Error: SLES12-SP3 does not contain the expected repository (/srv/tftpboot/suse-12.3/x86_64/install/content failed MD5 checksum)```

nicolasbock committed:
crowbar: Relax conditional for aborting installation (SCRD-8330)

The `install-chef-suse.sh` script is not only called during initial
deployment, but also during a Crowbar restore operation. During a
restore, the `crowbar-init.service` might not be running and the
current conditional testing for whether the installation can continue
is too strict in this case. This change relaxes the conditional by
unconditionally running the `reset_crowbar` step, while intercepting a
potential failure.

Signed-off-by: Nicolas Bock <nicolas.bock@suse.de>
(cherry picked from commit 32f8b7ac8a61e079454aae1ae9a7c83ef02243e7)

dirkmueller commented:
@toabctl I dropped the problematic commit

JanZerebecki commented:
This branch is soon EOL, so this probably shouldn't be merged any more. Please reopen if you feel otherwise.
